Source/WebCore/loader/HistoryController.h

```diff
@@ -94,12 +94,14 @@ class HistoryController final : public CanMakeCheckedPtr<HistoryController> {
 
     void setDefersLoading(bool);
 
+    Ref<HistoryItem> createNewItem(HistoryItemClient&, RefPtr<DocumentLoader>);
```

Comment:
Also maybe rename this function to overload `HistoryController::createItem`? Or name `createItemWithLoader` or something.